SU-SOE/soe_basic
# D8CORE-2446: Setting up hero banners for SOE

# READY FOR REVIEW

# Summary
- Changing display for the hero banner. On all mobile views the text should appear on the gradient.

# Needed By (Date)
- Friday

# Urgency
- Med

# Steps to Test

1. Pull in this branch
2. Build
3. Add a hero banner to a page. Verify it matches the designs in the ticket.
https://stanfordits.atlassian.net/browse/D8CORE-2446

# Affected Projects or Products
- SOE Subtheme

# Associated Issues and/or People
- D8CORE-2446

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
